.circleci/check-cpp-format.sh

```diff
@@ -0,0 +1,175 @@
+#!/bin/bash
+#
+# Copyright 2020-present Open Networking Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+set -ex
+
+# Files in this branch that are different from master.
+CHANGED_FILES=$(git diff-tree --no-commit-id --name-status -r HEAD origin/master \
+                  | grep -v "^D" \
+                  | cut -f 2 \
+                  | grep -F -e ".cc" -e ".h" || :
+              )
+
+# List of files that are already formatted.
+read -r -d '\0' KNOWN_FILES << EOF
+stratum/glue/gtl/cleanup_test.cc
```

Comment:
Are there supposed to be so many duplicates?

Reply:
no, copy&paste fail